modules/auxiliary/dos/http/webkitplus.rb

```diff
@@ -0,0 +1,59 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
```

Comment:
This also needs to
```ruby
include Msf::Auxiliary::Dos
```
to avoid being unintentionally run by Pro and other tools.